README.md

```diff
@@ -1,13 +1,40 @@
 ![badge-img](https://img.shields.io/badge/Made%20at-%23AstroHackWeek-8063d5.svg?style=flat)
 # stateoftheuniverse
-Returns the state of the universe
-An Astro Hack Week 2019 project.
 
-Proposed by Mariarosa Marinelli
+## A python package that returns the current state of the universe
 
-And made happen by:
-- Mariarosa Marinelli[![github icon](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/22/github.png)](https://github.com/astromariarosa/)
-- Ania Kapuścińska[![github icon](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/22/github.png)](https://github.com/lambdanis/)
-- Timothy Gebhard[![github icon](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/22/github.png)](https://github.com/timothygebhard/)
-- Eli Prater[![github icon](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/22/github.png)](https://github.com/pratere/)
-- Oliver Hall[![github icon](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/22/github.png)](https://github.com/ojhall94/)
+Would you like to know when the phase of the moon? What about the current tally of exoplanets and asteroids? Curious about the constellations and planets you can see in the sky tonight? Want to celebrate the birthdays of historical astronomers?
```

Comment:
"Would you like to know when the phase of the moon?" Did you mean to say "Would you like to know which phase the moon is in?"